### ISSUE TITLE
Clusternator adds code to project - can this be in a package instead?

### ISSUE THREAD

**Issue by SethDavenport** [rangle/the-clusternator#267]

**Comment by bennett000** [comment]
@SethDavenport there are definitely a number of ways we could avoid polluting codebases.  One solution is to just have `clusternator` do everything the scripts do.  Another would be some sort of "light"/ci version which is I think where you're going with the `clusternator-deps` package.

I would prefer to go with the light/ci solution since it involves only an `aws-sdk` dependency.

**Comment by SethDavenport** [comment]
Yeah that's exactly what I mean - I don't want to make people depend on the CLI; we could instead bundle these scripts into a 'clusternator-runtime' sort of idea.  Except it's build time...

**Comment by bennett000** [comment]
That's a really good solution, especially since it allows versioning.  Something like `clusternator-ci-aws` might work for the first set

**Comment by SethDavenport** [comment]
Lower priority than other stuff for now though I'd say.

**Comment by SethDavenport** [comment]
* [ ] adjust `clusternator init` to do an `npm install --save-dev clusternator-runtime`
* [ ] publish new version of clusternator to npm (0.7?)
* [ ] deploy new clusternator server
